src/main/scala/com/typesafe/sbt/bundle/SbtBundle.scala

```diff
@@ -181,13 +180,22 @@ object Import {
   case class Endpoint(
     bindProtocol: String,
     bindPort: Int,
-    @deprecated("services are deprecated in favour of ACLs", since = "1.3.0") services: Option[Set[URI]],
+    services: Option[Set[URI]],
     serviceName: Option[String],
     acls: Option[Set[RequestAcl]])
 
+  object BundleConfVersions extends Enumeration {
+    val V_1_1_0 = Value("1.1.0")
```

Comment:
I think we have it as version `1.1.0` - https://github.com/sbt/sbt-bundle/pull/62/files#diff-6d18a5d70eaf719443d1015a15137257L653 - so because of this, the version is increased to `1.2.0`